app/lib/common/widgets/attachments/attachment_options.dart

```diff
@@ -26,31 +27,27 @@ class AttachmentOptions extends StatelessWidget {
         if (!isDesktop)
           ListTile(
             onTap: () {
-              Navigator.of(context).pop();
```

Comment:
Navigator.of(context).pop();

It should be here as it was before and need to have standalone route control of `AttachmentOptions`  itself instead of putting that one different class (Ex `attachmentConfirmationDialog`)

Reply:
Fixed by 0a89376